frontend/src/metabase/setup/components/CompletedStep/CompletedStep.tsx

```diff
@@ -24,15 +29,30 @@ export const CompletedStep = (): JSX.Element | null => {
 
   const baseUrl = window.MetabaseRoot ?? "/";
 
+  const handleClick = () => {
+    if (checkboxValue && email) {
```

Comment:
`&& email` is mostly to make ts happy, but also because it doesn't hurt, it makes no sense to subscribe without an email anyway